scania-digital-design-system/sdds
# feat(stepper): webcomponent v2

**Describe pull-request**  
Version 2 of the stepper webcomponent. This one has `<li>` as direct child of `<ol>` for a11y. This means exposing props for the sdds-stepper-item that the user should never really interact with, not sure what is the best approach here.

**Solving issue**  
Fixes: -

**How to test**  
1. **Checkout branch and run storybook**
2. Check in Components -> Stepper -> Web Component
3. Check that the style is aligned with figma
4. Check out the render method of the sdds-stepper.

**Suggested test steps**

**Screenshots**  
-

**Additional context**  
-
